app/src/pages/inside/historyPage/historyView/historyTable/historyItemsGrid/historyItem/img/comment-inline.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
```

Comment:
Now we decided to store all static images and icons in one place: /src/common/img/. Please replace all icons added in that PR.